### DISCUSSION TITLE
KAZTDT1

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#381] [Show and tell]
### Channel ID

KAZTDT1.us

### Stream URL

https://live.mybtv.net/manifest/4f0f25b338c3293216e8b36b00f3248b08be929d/live-springserve/cf237858-3250-4f41-855e-f4ce471dd48c/0.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://live.mybtv.net/manifest/4f0f25b338c3293216e8b36b00f3248b08be929d/live-springserve/cf237858-3250-4f41-855e-f4ce471dd48c/0.m3u8 (offline)
<img width="832" alt="image" src="https://user-images.githubusercontent.com/7253922/232255733-5692f4ab-310f-4963-9e59-2627c80326e4.png">
